.tekton/determine-image-tag-task.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: determine-image-tag
+  namespace: rh-acs-tenant
+spec:
+  description: Determining image tag with custom logic.
+  params:
+  results:
+  - name: image-tag
+    description: Image Tag determine by custom logic.
+  steps:
+  - name: determine-image-tag
+    image: registry.access.redhat.com/ubi8:latest
+    script: |
+      #!/usr/bin/env bash
+      set -euo pipefail
+      dnf -y upgrade --nobest
+      dnf -y install git make
+      cd "$(workspaces.source.path)/source"
+      echo -n "$(make --quiet --no-print-directory tag)-fast" | tee "$(results.image-tag.path)"
```

Comment:
I just realized that we can also run `scripts/konflux/fail-build-if-git-is-dirty.sh` to actually prevent `-dirty` suffixes on the output.

Reply:
Added in https://github.com/stackrox/stackrox/pull/10749/commits/fe9f69ba3b36e372b39bd5d853a02889644502b3 before the `make tag` call.